clintrials/dosefinding/efftox.py

```
@@ -146,8 +147,94 @@ def _L_n(D, mu_T, beta_T, mu_E, beta1_E, beta2_E, psi):
+    """
+    limits = [(dist.ppf(epsilon), dist.ppf(1 - epsilon)) for dist in priors]
+
+    lik_integrand = (
+        lambda x: _L_n(cases, x[:, 0], x[:, 1], x[:, 2], x[:, 3], x[:, 4], x[:, 5])
+        * priors[0].pdf(x[:, 0])
+        * priors[1].pdf(x[:, 1])
+        * priors[2].pdf(x[:, 2])
+        * priors[3].pdf(x[:, 3])
+        * priors[4].pdf(x[:, 4])
+        * priors[5].pdf(x[:, 5])
+    )
```

Ruff (0.15.18)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `lik_integrand` as a `def`

(E731)